tiledb/bioimg/converters/base.py

```diff
@@ -136,6 +142,7 @@ def to_tiledb(
         level_min: int = 0,
         tiles: Mapping[str, int] = {},
         preserve_axes: bool = False,
+        **register_kwargs: Mapping[str, str],
```

Comment:
I had in mind `register_kwargs` without the `**`, otherwise it's still a flat list of keywords from the caller perspective.